client/state/help/ticket/actions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+
+import {
+	HELP_TICKET_CONFIGURATION_REQUEST,
+	HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS,
+	HELP_TICKET_CONFIGURATION_REQUEST_FAILURE,
+	HELP_TICKET_CONFIGURATION_DISMISS_ERROR,
+} from 'state/action-types';
+
+export const ticketSupportConfigurationRequestSuccess = ( configuration ) => {
+	return {
+		type: HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS,
+		...configuration,
```

Comment:
I just noticed that if the API returned a JSON response containing a `type` property, it would override the action type and could theoretically execute any Redux action. It's unlikely to be a security risk, but seems like something we should fix either way.